build-logic/buildquality/src/main/kotlin/gradlebuild.incubation-report.gradle.kts

```diff
@@ -23,6 +22,7 @@ import gradlebuild.incubation.tasks.IncubatingApiReportTask
 
 plugins {
     java
+    groovy
```

Comment:
🤔 Did we not have the `groovy` plugin applied before?

Reply:
Well, we always the Groovy plugin applied. In order to obtain the extension function, we need to apply it again. This worked before since the Groovy plugin was applied when we tried to obtain the groovy source set directory convention.